core/services/feeds/service.go

```diff
@@ -362,7 +362,7 @@ func (s *service) ApproveJobProposal(ctx context.Context, id int64) error {
 		return errors.Wrap(err, "could not generate job from spec")
 	}
 
-	err = postgres.NewQ(s.queryer, postgres.WithParentCtx(ctx)).Transaction(func(tx postgres.Queryer) error {
+	err = postgres.NewQ(s.queryer, postgres.WithParentCtx(ctx)).Transaction(s.lggr, func(tx postgres.Queryer) error {
```

Comment:
Perhaps we want to inject the logger onto the Q instead?

Reply:
I don't think that is necessary to replace any existing helper func calls, but yeah it could be useful.